Doc/library/shutil.rst

```diff
@@ -343,8 +343,7 @@ Directory and files operations
 
    Return disk usage statistics about the given path as a :term:`named tuple`
    with the attributes *total*, *used* and *free*, which are the amount of
-   total, used and free space, in bytes.  On Windows, *path* must be a
-   directory; on Unix, it can be a file or directory.
```

Comment:
Nit: I'd keep the "it can be a file or directory." part and add a versionchanged marker to explain that it now can accept file and directory on Windows. Example:

```rst
.. versionchanged:: 3.8
   Explain the behaivor change here.
```

Reply:
Ok - cool - I wasn't sure what the convention was. I'll update.